packages/admin/cms-admin/src/generator/future/generateGrid.ts

```diff
@@ -508,6 +508,18 @@ export function generateGrid(
                             .join(",\n")} ], `
                       : ""
               }
+              ${
+                  config.initialFilter
+                      ? `initialFilter:{ ${
+                            config.initialFilter.linkOperator ? `linkOperator: "${config.initialFilter.linkOperator}" as GridLinkOperator,` : ""
```

Comment:
why is this cast needed? couldn't you use `GridLogicOperator.Or`?

Reply:
As I use just strings as values at the moment, I have to cast them as GridLinkOperator, to match the typing.

Reply:
an alternative without as cast could be:
`linkOperator: GridLinkOperator.${config.initialFilter.linkOperator == "or" ? "Or" : "And"}`

generally, `as` casts should be avoided, they may hide type issues.